### PR TITLE
Temporarily deleting the 'Guides' link from the home page

### PR DESCRIPTION
**Pull request by gilardh**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ function gtag() {
                 <li><a href="#benefits" class="smooth-scroll"><?php L::W("Benefits"); ?></a></li>
                 <li><a href="#exchange" class="smooth-scroll"><?php L::W("Exchange"); ?></a></li>
                 <li><a href="#wallets" class="smooth-scroll"><?php L::W("Wallets"); ?></a></li>
-                <li><a href="http://bitcoinzguiding.ovh" class="smooth-scroll" target="_blank"><?php L::W("Guides"); ?></a></li>
+                <!-- <li><a href="http://bitcoinzguiding.ovh" class="smooth-scroll" target="_blank"><?php L::W("Guides"); ?></a></li> -->
```
Comment (egyptianbman) [inline index.php]:
```suggestion
```

File: index.php [hunk 2]
```diff
@@ -126,7 +126,7 @@ function gtag() {
                         <li><a href="#benefits" class="smooth-scroll"><?php L::W("Benefits"); ?></a></li>
                         <li><a href="#exchange" class="smooth-scroll"><?php L::W("Exchange"); ?></a></li>
                         <li><a href="#wallets" class="smooth-scroll"><?php L::W("Wallets"); ?></a></li>
-                        <li><a href="http://bitcoinzguiding.ovh" class="smooth-scroll" target="_blank"><?php L::W("Guides"); ?></a></li>
+                        <!-- <li><a href="http://bitcoinzguiding.ovh" class="smooth-scroll" target="_blank"><?php L::W("Guides"); ?></a></li>-->
```
Comment (egyptianbman) [inline index.php]:
```suggestion
```

**Review by egyptianbman — CHANGES_REQUESTED** [review]
It looks like the domain is completely gone. let's go ahead and remove the link  altogether, please.

**Review by egyptianbman — APPROVED** [review]
Thank you!

**Comment by gilardh** [comment]
Thanks to you too.

**Pull request merged**